tensilelite/Tensile/KernelWriter.py

```diff
@@ -2599,14 +2609,6 @@ def kernelBody( self, kernel, tensorParametersA, tensorParametersB ):
       if (kernel["DirectToVgprA"] or kernel["DirectToVgprB"] or kernel["DirectToLdsA"] or kernel["DirectToLdsB"]):
         mEnd = kernel["DepthU"]//(kernel["MatrixInstK"]*kernel["LocalSplitU"])
 
-      # TailLoop unroll case (mEnd > 1), we need to keep these vgpr
-      if mEnd == 1:
-        # add vgprBuffer for local read to vgprPool because we won't issue local read in this section
-        self.vgprPool.add(self.states.a.startVgprValu , \
-          self.states.lastValuAB - self.states.a.startVgprValu, "ValuAB") # Add as available
-        module.addComment1("Tail: add ValuA/B vgpr buffer [%u...%u) to pool" % \
-                          (self.states.a.startVgprValu, self.states.a.startVgprValu+self.states.lastValuAB))
-
```

Comment:
This is for **DTV will use the Valu VGPRs which is released in the beginning of tail loop**. @Serge45 @solaslin Please check this part of tail loop because you are engaging to Swizzle+DTV.